docs/knowledge_base/threat_feed.md

```diff
@@ -97,3 +99,42 @@ For example, if you want to limit the items per page to 3 since July 19, 2023 yo
 ```text
 https://threats.phylum.io/?per_page=3&since=2023-07-19
 ```
+
+## Cursor Based Pagination
+The threat feed supports cursor based pagination. By providing the `cursor` parameter you can iterate through the data in a guaranteed order. For example:
```

Comment:
It might be worth adding a table of query options, like is done in the "Additional Query Options" section. Otherwise, it is kind of confusing to know what, if anything, is meant to go with "providing the `cursor` parameter" or _could_ be provided.